internal/latestrelease/finder.go

```diff
@@ -45,14 +46,25 @@ func versionFromTag(release *github.RepositoryRelease, toolName string) string {
 	return release.GetTagName()
 }
 
+const (
+	mongoCLI = "mongocli"
+	atlasCLI = "atlascli"
+)
```

Comment:
drive by these were being imported from the `version` package but only used here